velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -346,11 +346,15 @@ class VectorFuzzer {
   FuzzerGenerator rng_;
 };
 
-/// Generates a random type, including maps, vectors, and arrays. maxDepth
+/// Generates a random type, including maps, rows, and arrays. maxDepth
 /// limits the maximum level of nesting for complex types. maxDepth <= 1 means
 /// no complex types are allowed.
 TypePtr randType(FuzzerGenerator& rng, int maxDepth = 5);
 
+/// Same as the function above, but only generate orderable types, maps would
+/// not be generated since it is not orderable.
+TypePtr randOrderableType(FuzzerGenerator& rng, int maxDepth = 5);
```

Comment:
Let's break up this PR into smaller parts. First piece could be this change in VectorFuzzer + test.

Reply:
Got it, will create a new PR to add randOrderableType in  VectorFuzzer + test, and remove them from this one.